models/user.go

```diff
@@ -162,6 +162,9 @@ type User struct {
 	DiffViewStyle       string `xorm:"NOT NULL DEFAULT ''"`
 	Theme               string `xorm:"NOT NULL DEFAULT ''"`
 	KeepActivityPrivate bool   `xorm:"NOT NULL DEFAULT false"`
+
+	// OAuth
+	OAuthProvider       int64
```

Comment:
```suggestion
	OAuthProviderID       int64
```

You will also need a migration for this - as existing users won't have any.

Reply:
We have an `external_account` table, I think it's already ready.